releasenotes/notes/fix-restore-dict-int-keys-c56f8f9bd0279aca.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    :class:`qiskit.providers.ibmq.runtime.RuntimeDecoder` can now restore dictionary integer keys
+    in optimizer settings from a JSON string representation dumped by the
+    :class:`qiskit.providers.ibmq.runtime.RuntimeEncoder`.
```

Comment:
The cross reference links need to be fixed

Reply:
Thanks. I'll fix all the links after the migration PR merges and when I prepare the release.